convert_fendl.py

```diff
@@ -41,10 +41,10 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
                     default='earliest', help="Output HDF5 versioning. Use "
                     "'earliest' for backwards compatibility or 'latest' for "
                     "performance")
-parser.add_argument('-r', '--release', choices=['3.1d', '3.1a', '3.0', '2.1'],
-                    default='3.1d', help="The nuclear data library release "
-                    "version. The currently supported options are "
-                    "3.1d, 3.1a, 3.0 and 2.1")
+parser.add_argument('-r', '--release', choices=['3.2','3.1d', '3.1a', '3.0',
+                    '2.1'],  default='3.1d', help="The nuclear data library "
```

Comment:
Probably makes sense to set the default to 3.2, right?
```suggestion
                    '2.1'],  default='3.2', help="The nuclear data library "
```